src/rut.directive.ts

```diff
@@ -1,31 +1,29 @@
-import { Directive, Renderer, ElementRef, EventEmitter, Output } from '@angular/core';
+import {Directive, EventEmitter, Output} from '@angular/core';
 import * as rutHelpers from 'rut-helpers';
 
 @Directive({
-  selector: 'input[formatRut]',
+  selector: '[formatRut]',
   host: {
-    '(blur)': 'onBlur($event.target.value)',
-    '(focus)': 'onFocus($event.target.value)',
-    '(input)': 'onChange($event.target.value)',
+    '(blur)': 'onBlur($event)',
+    '(focus)': 'onFocus($event)',
+    '(input)': 'onChange($event)',
   },
 })
 export class RutDirective {
   @Output() public rutChange: EventEmitter<any>;
 
-  constructor(
-    private elementRef: ElementRef,
-    private renderer: Renderer) {
-      this.rutChange = new EventEmitter();
-    }
+  constructor() {
+    this.rutChange = new EventEmitter();
+  }
 
-  public onFocus(value: string) {
-    let cleanedRut: string = rutHelpers.rutClean(value);
-    this.renderer.setElementProperty(this.elementRef.nativeElement, 'value', cleanedRut);
+  public onFocus(ev: Event) {
+    let htmlInputElement = <HTMLInputElement>ev.target;
```

Comment:
expected variable-declaration: 'htmlInputElement' to have a typedef<br>missing whitespace